commands/narrator/day.js

```diff
@@ -2982,6 +2983,271 @@ module.exports = {
             }
         }, 4500)
 
+        //hacker
+        let THEHACKER = []
+        for (let z = 0; z < hacker.length; z++) {
+            let chan1 = message.guild.channnels.cache.get(hacker[z])
+            for (let y = 1; y < 17; y++) {
+                let tempguy = message.guild.members.cache.find((m) => m.nickname === y.toString())
+                if (tempguy) {
+                    THEHACKER = tempguy
```

Comment:
what is this line even doing? you're changing the array that was there into a GuildMember and then never using it?

Reply:
Basically the array is an mistake
But this whole thing is for the ss thing where it checks if the guy is in the array thekiller however i put the thing after ss check so its pretty pointless